pkg/crd/gen.go

```diff
@@ -312,6 +312,11 @@ func FindKubeKinds(parser *Parser, metav1Pkg *loader.Package) map[schema.GroupKi
 			switch namedField.Obj().Name() {
 			case "ObjectMeta":
 				hasObjectMeta = true
+				// Clear the godoc comment from the ObjectMeta field
+				// so it does not show up as the description, because
+				// description is not allowed in metadata of CRD structural schema.
+				field.Doc = ""
+				info.Fields[i] = field
```

Comment:
Hi, `info.Fields[i].Doc = ""` may be better.

Reply:
Thanks!